doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -185,14 +168,121 @@ Traceability to Reference Model.
 | `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. | |
 | `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. | |
 | `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. | |
-<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
-| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
 | `req.sec.ntw.03` | Networking | The Architecture **must** have the underlay network incorporate encrypted and/or private communications channels to ensure its security. | |
-|----|--------------|-----------------------------|-------|
 | `req.sec.ntw.04` | Networking | The Architecture **must** configure all of the underlay network components to ensure the complete separation from the overlay customer deployments. | |
 | `req.sec.ntw.05` | Networking | The Architecture **must** have the underlay network include strong access controls that adhere to the V1.1 NIST Cybersecurity Framework. | |
+<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
+
 <p align="center"><b>Table 2-8:</b> OpenStack Security Requirements.</p>
 
+
+
+<a name="2.4"></a>
+### 2.4 Architecture and OpenStack Recommendations
+
+The requirements listed in this section are optional, viz. do not need to be implemented in conformant implementations.
```

Comment:
```suggestion
The requirements listed in this section are optional, and are not required in order to be deemed a conformant implementation.
```
slight edit; not sure about the use of 'viz.' in this sentence.

Reply:
@kedmison Changed - Thanks